app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if is_logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end 
+  end
+
+  get '/tweets/new' do 
+    if !is_logged_in?
+      redirect '/login'
+    else 
+      erb :'/tweets/create_tweet'
+    end 
+  end
+
+  post '/tweets' do 
+    if !@params["content"].empty?
+      @tweet = Tweet.create(:content => @params["content"])
+      current_user.tweets << @tweet
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```